helm/install/values.yaml

```diff
@@ -2,23 +2,23 @@
 ## Provide image repository and tag
 image:
   repository: registry.developers.crunchydata.com/crunchydata
-  tag: ubi8-5.0.3-0
+  tag: ubi8-5.0.4-0
```

Comment:
See #57. Should be:

```yaml
image:
  image: registry.developers.crunchydata.com/crunchydata/postgres-operator:ubi8-5.0.4-0
```

Reply:
Rebased to pull in change and then updated to `5.0.4`.